### PR TITLE
added api support

### PR DESCRIPTION
**Pull request by dbashyal**
Feel free to merge and extend API support. I am now using your module to fetch new orders using API in node app. Currently I have done List and Update. Added new column 'is_new_order' as a flag to fetch only new orders.

### CODE REVIEW

File: app/code/community/ProxiBlue/OrderSyncQueRunner/etc/api.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <api>
+        <resources>
+            <order_sync_que_runner translate="title" module="ordersyncquerunner">
+                <model>ordersyncquerunner_resource/que_api</model>
+                <title>Order Sync Que Runner</title>
+                <acl>order_sync_que_runner</acl>
+                <methods>
+                    <list translate="title" module="ordersyncquerunner">
+                        <title>List New Order Increment Ids</title>
+                        <method>listNewOrdersQue</method>
```
Comment (erfanimani) [inline app/code/community/ProxiBlue/OrderSyncQueRunner/etc/api.xml:12]: Might be a good idea to start using the correct spelling of queue for the API ^^

**Comment by ProxiBlue** [comment]
At some point, yeah, my bad spelling needs to be fixed! when I have time

**Comment by erfanimani** [comment]
yeah, I meant to fix up the spelling for this PR so that integrations that are based on this won't have to change later.. cost of changing now is less than cost of changing later @dbashyal. Anyhow, all g :)

**Comment by ProxiBlue** [comment]
I dont use this currently. If one of you, or both want  admin on it to merge happy to do so.

**Comment by erfanimani** [comment]
Would be good, use it on a lot of sites

**Pull request merged**